llvm/test/Transforms/GlobalOpt/resolve-static-ifunc.ll

```diff
@@ -0,0 +1,81 @@
+; RUN: opt --passes=globalopt -o - -S < %s | FileCheck %s
+
+target datalayout = "e-m:e-i8:8:32-i16:16:32-i64:64-i128:128-n32:64-S128"
+target triple = "aarch64-unknown-linux-gnu"
+
+$callee_with_trivial_resolver.resolver = comdat any
+@callee_with_trivial_resolver.ifunc = weak_odr dso_local ifunc void (), ptr @callee_with_trivial_resolver.resolver
+define weak_odr ptr @callee_with_trivial_resolver.resolver() comdat {
```

Comment:
Are the comdats in this test relevant somehow? They don't seem to be.

Reply:
Don't need them. They're there because that's what the frontend adds for `__attribute__((target_version(...)))` functions.